deploy/helm/flightctl/templates/api/flightctl-api-networkpolicy.yaml

```
@@ -0,0 +1,23 @@
+    {{- include "common.labels.standard" $ | nindent 4 }}
+    app.kubernetes.io/component: api
+spec:
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+          network.openshift.io/policy-group: ingress
+  podSelector:
+    matchLabels:
+      {{- include "common.labels.matchLabels" $ | nindent 6 }}
+      app.kubernetes.io/component: api
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)